.github/workflows/databases.yml

```diff
@@ -40,6 +38,9 @@ jobs:
           tools: composer:v2
           coverage: none
 
+      - name: Set Framework version
+        run: composer config version "10.x-dev"
```

Comment:
Set `version` to `composer.json` would allow us to explicitly declare `laravel/framework` will resolved to `10.x-dev` and this allows `laravel/prompts` to be installed instead of using `fetch-depth` (retrieve all branches and tags from Git).